### PR TITLE
Preview

### LINKED ISSUE #8: Create input fields using react hooks
Input fields should be created for:
- email subject
- authkey/credentials
- an option to choose a CSV file

### LINKED ISSUE #13: Add a preview window
- Include a preview window that shows the rendered email body output.
- The output should be rendered HTML (if the input of the email body is in HTML) and not in plain text.

refer to the wireframe(mentioned in the [notion guide](https://grass-text-1a7.notion.site/Maildart-Frontend-5ea6f3ea336a487da18cd2a99a21a213)) and size the preview window accordingly

### PR DESCRIPTION
**Pull request by Rachel-glitch**
This is my solution to fix issue: #13

### CODE REVIEW

**Review by SwarupKharul — APPROVED** [review]

**Review by Likkiii — CHANGES_REQUESTED** [review]

**Review by Likkiii — CHANGES_REQUESTED** [review]

**Review by Likkiii — CHANGES_REQUESTED** [review]

File: src/components/TextArea.js [hunk 1]
```diff
@@ -3,28 +3,90 @@ import React, { useState } from 'react';
...
+                className=' m-2 w-11/12 h-10 border p-0.5 rounded-md'
+            />
+
+            <h1 className='text-white m-2'>Recipients File</h1>
+            <div className='bg-white m-2 w-11/12 h-10 border p-0.5 rounded-md flex items-end justify-end'>
+            <label
+                class=" mr-3 text-black px-4 py-1 bg-gray-400 rounded-md shadow-md tracking-wide  cursor-pointer ease-linear ">
+                <i class="fas fa-cloud-upload-alt fa-3x"></i>
+                <span class="mt-2 text-base leading-normal">Choose file</span>
+                <input type='file' value ={file} class="hidden" />
```
[the remark this answers is not in the document]
Comment (Rachel-glitch) [reply src/components/TextArea.js]: I made changes, but I don't know if you can see them.

File: src/components/TextArea.js [hunk 2]
```diff
@@ -1,33 +1,76 @@
...
+          <button class="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4  rounded  ">
+            SEND
+          </button>
+        </div>
+      </div>
+      <div className="flex-1">
+        <div className="Notes m-2 mt-10 bg-gray-100 w-11/12 h-auto text-gray-400  px-4  pb-12 text-align: center; ">
+          <div className="EmailText m-2 mt-0 bg-white h-auto text-black py-2 px-4">
+            {emailText}
+          </div>
```
Comment (Likkiii) [inline src/components/TextArea.js:64]: Great work!
Just one final change.
The text being entered is not visible in the preview window, please check.

**Review by Likkiii — CHANGES_REQUESTED** [review]

File: src/components/TextArea.js (hunk 2 above)
Comment (Rachel-glitch) [reply src/components/TextArea.js:64]: now, when you click on button SEND, text will be visible in preview window

**Review by Likkiii — APPROVED** [review]

**Comment by Likkiii** [comment]
LGTM
Well done @Rachel-glitch!

**Pull request merged**